boards/arm/stm32f769i_disco/stm32f769i_disco.dts

```diff
@@ -22,6 +22,11 @@
 		zephyr,flash-controller = &mx25l51245g;
 	};
 
+	sdram1: sdram@c0000000 {
+		device_type = "memory";
+		reg = <0xc0000000 DT_SIZE_M(16)>;
```

Comment:
According to board manual SDRAM embedded is 128M, any reason to set 16M only ?

Reply:
128-Mbit = 16M bytes, right ?

Reply:
Right :-), too early review..